modules/machineset-yaml-rhv.adoc

```diff
@@ -113,16 +114,18 @@ If you are using a version earlier than {rh-virtualization} 4.4.8, see link:http
+<17> Optional: Specify the UUID of the storage domain for the compute node's disks. If none is provided, the compute node is created on the same storage domain as the control nodes. (default)
 
-<18> Optional: Specify the vNIC profile ID.
+<18> Optional: List of the network interfaces of the VM. If you include this parameter, {product-title} discards all network interfaces from the template and creates new ones.
 
-<19> Specify the name of the secret that holds the {rh-virtualization} credentials.
+<19> Optional: Specify the vNIC profile ID.
 
-<20> Optional: Specify the workload type for which the instance is optimized. This value affects the `{rh-virtualization} VM` parameter. Supported values: `desktop`, `server` (default), `high_performance`. `high_performance` improves performance on the VM, but there are limitations. For example, you cannot access the VM with a graphical console. For more information, see link:https://access.redhat.com/documentation/en-us/red_hat_virtualization/4.4/html-single/virtual_machine_management_guide/index#Configuring_High_Performance_Virtual_Machines_Templates_and_Pools[Configuring High Performance Virtual Machines, Templates, and Pools] in the _Virtual Machine Management Guide_.
-<21> Optional: AutoPinningPolicy defines the policy that automatically sets CPU and NUMA settings, including pinning to the host for this instance. Supported values: `none`, `resize_and_pin`. For more information, see link:https://access.redhat.com/documentation/en-us/red_hat_virtualization/4.4/html-single/virtual_machine_management_guide/index#Setting_NUMA_Nodes[Setting NUMA Nodes] in the _Virtual Machine Management Guide_.
-<22> Optional: Hugepages is the size in KiB for defining hugepages in a VM. Supported values: `2048` or `1048576`. For more information, see link:https://access.redhat.com/documentation/en-us/red_hat_virtualization/4.4/html-single/virtual_machine_management_guide/index#Configuring_Huge_Pages[Configuring Huge Pages] in the _Virtual Machine Management Guide_.
-<23> Optional: A list of affinity group names that should be applied to the VMs. The affinity groups must exist in oVirt.
+<20> Specify the name of the secret that holds the {rh-virtualization} credentials.
+
+<21> Optional: Specify the workload type for which the instance is optimized. This value affects the `{rh-virtualization} VM` parameter. Supported values: `desktop`, `server` (default), `high_performance`. `high_performance` improves performance on the VM, but there are limitations. For example, you cannot access the VM with a graphical console. For more information, see link:https://access.redhat.com/documentation/en-us/red_hat_virtualization/4.4/html-single/virtual_machine_management_guide/index#Configuring_High_Performance_Virtual_Machines_Templates_and_Pools[Configuring High Performance Virtual Machines, Templates, and Pools] in the _Virtual Machine Management Guide_.
```

Comment:
Per the ISG:

> Generally, when you specify the name of a command, parameter, or option in text, specify the word command, parameter, or option after the name.
Command, parameter, and option names are not translated; without the identifying noun, users might be confused about what the name represents.

and 

> Avoid using expletive constructions such as it is, there are, and there is, which hide the subject of the sentence.

Reply:
New PR to handle the updates - https://github.com/openshift/openshift-docs/pull/45224